osu-cass/cass-pelican
# Add FEATURED variable

Fixes slide show so that it only displays "featured" articles. Should be merged alongside https://github.com/osuosl/dougfir-pelican-theme/pull/50.

## Testing this PR
1. Make sure that ``:category: featured-stories`` is set on selected articles
2. ``cd dougfir-pelican-theme/``
3. ``git checkout thai/slideshow_categories``
4. Navigate back to cass-pelican
5. (``make clean``), ``make html``, ``make devserver``
6. Visit localhost:8000 - the slideshow should now only display the "featured" pages 

@Kennric @kelnera
